src/tests/test_properties/test_properties.cpp

```diff
@@ -30,7 +30,7 @@ extern "C" {
 }
 #include <cfloat>
 
-static const bool kRunLongTests = true;
+static const bool kRunLongTests = false;
```

Comment:
This should not be changed.

Reply:
Good catch. In another file I also see some white space changes that are not required. I will fix those up.